assets/package.json

```diff
@@ -2,6 +2,38 @@
   "name": "console",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
+    "start": "HTTPS=true react-app-rewired start",
+    "build": "react-app-rewired build",
+    "postbuild": "cpx \"build/**\" ../priv/static --clean",
+    "test": "react-app-rewired test",
+    "lint": "eslint ./src",
+    "fix": "eslint ./src --fix",
+    "eject": "react-scripts eject",
+    "graphql-codegen": "graphql-codegen --config codegen.yml"
+  },
+  "eslintConfig": {
+    "extends": "react-app",
```

Comment:
FYI, not necessary here, but we have our own lint config `eslint-config-pluralsh`

Reply:
I have created ENG-254 for that because it might need some updates.